### DISCUSSION TITLE
How can I compile my foundry sol code into bytecode?

### DISCUSSION THREAD

**Discussion by utx0** [foundry-rs/foundry#4753] [Q&A] [answered]
How can I compile my foundry sol code into bytecode?

**Comment by DaniPopes** [comment]
After you build a Foundry project with `forge build` you will find the compiler output for each file under `out/Name.sol/Name.json`, which includes the ABI, bytecode among others.

**Comment by utx0** [comment, marked as answer]
I actually had to run this to get the ABI and BIN files generated: 

`forge build --extra-output-files abi bin`
